db/migrate/20211112180117_add_confirmable_fields_to_user.rb

```diff
@@ -1,6 +1,6 @@
 class AddConfirmableFieldsToUser < ActiveRecord::Migration[6.1]
   def change
-    change_table :users do |t|
+    change_table :users, bulk: true do |t|
```

Comment:
I don't think we should be modifying already existing migrations since there is a chance that the already merged migration has already been run either on prod or some other devs computer, meaning that they will never get this change on their DB because this migration has already been run

Reply:
How can we do for an existing migration raising rubocop errors?

Reply:
That's a fair point! 
In this case, given that the bulk keyword is only to be run on the migration at hand, this change is alright. If something like this happens in the future and it gets to affect the database schema we should do on separate migrations